packages/chart/src/ChartUtils.ts

```diff
@@ -185,187 +184,6 @@ class ChartUtils {
 
```

Comment:
I had a thought. Right now we're doing `new ChartUtils(dh)` in a bunch of places; which could result in multiple `ChartUtils` objects that are pretty much the same. Not the biggest deal, _but_, we _could_ provide a static method to get an existing chart utils object based on the API... and just call `ChartUtils.get(dh)` (or a util function `getChartUtils(dh)`) instead. Could use [WeakMaps](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/WeakMap#caching) to they get cleaned up automatically if the API gets garbage collected as well.

Probably wouldn't change much in terms of overhead, but thought I'd mention it.